app/views/cloud_volume/edit.html.haml

```diff
@@ -1,18 +1 @@
-%form#form_div{:name           => "angularForm",
-               'ng-controller' => "cloudVolumeFormController as vm",
-               "ng-show"       => "vm.afterGet",
-               "ng-cloak"      => "",
-               "miq-form"      => true,
-               "model"         => "vm.cloudVolumeModel",
-               "model-copy"    => "vm.modelCopy"}
-  = render :partial => "layouts/flash_msg"
```

Comment:
This render needs to stay (in both edit and new), otherwise the form can not render flash messages.